x/ibc-account/keeper/relay.go

```diff
@@ -40,7 +40,10 @@ func (k Keeper) TryRegisterIBCAccount(ctx sdk.Context, sourcePort, sourceChannel
 		Data: salt,
 	}
 
-	timeoutHeight := clienttypes.ZeroHeight()
```

Comment:
@colin-axner changing this to clienttypes.ZeroHeight() seems to cause the relayer to crash with a nondescript error

Reply:
This is probably a bug in the relayer. Lets open an issue and fix it

Reply:
see [relayer issue](https://github.com/cosmos/relayer/issues/481)

Reply:
It should be noted that hardcoding `revision height` will break for any chain without `-2` as a suffix

Reply:
otherwise any chain-ids with a revision numebr > 2 will timeout automatically

Reply:
Wdyt https://github.com/cosmos/relayer/pull/483